tests/bluetooth/test_emulation.py

```diff
@@ -62,6 +62,15 @@ async def setup_device(websocket, context_id):
             }
         })
 
+async def disable_simulation(websocket, context_id):
+    await execute_command(
+        websocket, {
+            'method': 'bluetooth.disableSimulation',
+            'params': {
+                'context': context_id,
+            }
+        })
+
 
```

Comment:
Please add the following tests:
* `bluetooth.disableSimulation` disables simulation (we should assert the simulation is really disabled)
* `bluetooth.disableSimulation` disables simulation only in the provided context (it will fail for now, so mark it as `pytest.xfail` until the simulation per context is supported)
* `bluetooth.disableSimulation` twice

Reply:
Done.

Thanks for the suggestion! I also updated those to remove `pytest.xfail` on old headless mode since old headless can work with bluetooth now (after https://crrev.com/c/6165347)